Mathlib/Analysis/InnerProductSpace/Defs.lean

```diff
@@ -10,9 +10,9 @@ import Mathlib.Data.Complex.Basic
 /-!
 # Inner product spaces
 
-This file defines inner product spaces.  We do not formally define Hilbert spaces, but they can be
-obtained using the set of assumptions
-`[NormedAddCommGroup E] [InnerProductSpace 𝕜 E] [CompleteSpace E]`.
+This file defines inner product spaces. For Hilbert spaces `HilbertSpace` is a typeclass alias, so
+that one can write `variable? [HilbertSpace 𝕜 E]` and get suggested
+`[RCLike 𝕜] [NormedAddCommGroup E] [InnerProductSpace 𝕜 E] [CompleteSpace E]`.
```

Comment:
Maybe
```suggestion
This file defines inner product spaces.
Hilbert spaces can be obtained using the set of assumptions
`[RCLike 𝕜] [NormedAddCommGroup E] [InnerProductSpace 𝕜 E] [CompleteSpace E]`.
For convenience, a variable alias `HilbertSpace` is provided so that one can write `variable? [HilbertSpace 𝕜 E]` 
and get this as a suggestion.
```